cloud/deploy/deploy.go

```diff
@@ -217,12 +217,15 @@ func Deploy(deployInput InputDeploy, client astro.Client, coreClient astrocore.C
 	}
 
 	dagFiles := fileutil.GetFilesWithSpecificExtension(dagsPath, ".py")
+	fmt.Println("deployment id:")
+	fmt.Println(deployInput.RuntimeID)
```

Comment:
are these fmt.println lines here debugging or on purpose?
there's quite a few added in this function

Reply:
still curious

Reply:
yes I'll remove